lib/src/charts/widgets/bar_chart.dart

```diff
@@ -27,6 +27,12 @@ class BarChart extends StatefulWidget {
   /// Defaults to true.
   final bool autoAnimate;
 
+  /// The Duration for which the chart should animate
+  final Duration? animDuration;
+
+  /// A custom Animation controller which can be provided by the user
+  final AnimationController? animation;
```

Comment:
`which can be provided by the user` makes this sound like this is internal documentation.
```suggestion
  /// A custom Animation controller to drive the animation
  final AnimationController? animation;
```